kivy/core/window/__init__.py

```diff
@@ -933,6 +933,33 @@ def _set_top(self, value):
     the position set in :class:`~kivy.config.Config`.
     '''
 
+    def _get_opacity(self):
+        return self._get_window_opacity()
+
+    def _set_opacity(self, opacity):
+        self._set_window_opacity(opacity)
+
+    def _get_window_opacity(self):
+        Logger.warning('Window: opacity is not implemented'
+                       ' in the current window provider.')
+
+    def _set_window_opacity(self, opacity):
+        Logger.warning('Window: opacity is not implemented'
```

Comment:
Maximize available width for the code:
```python
Logger.warning('Window: Opacity is not implemented in the current '
               'window provider')
```

Do the same in the `_get_window_opacity` method.